latex/Research_Strategy.tex

```diff
@@ -368,74 +416,73 @@
 
 
-{\bf Subaim 2.1: Cucurbituril-based receptors as model binding systems}
+\textbf{Subaim 2.1: Cucurbituril-based receptors as model binding systems}
 %Isaacs science
 
-\textbf{Cucurbituril derivatives for host-guest binding.} 
-%DLM: A bit odd to have the OA section start with a history header and this one not. May need to address.
-Building on previous success with cucurbit[$n$]uril (abbreviated CB[$n$]) experiments for SAMPL challenges~\cite{ma_acyclic_2012-2, cao_absolute_2014, she_glycoluril-derived_2016}, we will conduct a series of new experiments on these receptors for five new challenges,
- with experimental work conducted by co-investigator Isaacs, an expert on these systems who provided data for previous SAMPL challenges.
-CB[$n$] receptors are particularly well suited to our goals because they exhibit: (1) high binding affinities toward suitable guests in water comparable to protein-ligand affinities (routinely $\mu$M to nM; occasionally pM to fM)~\cite{cao_attomolar_2014, liu_cucurbituril_2005, mock_structure_1986, assaf_cucurbiturils:_2015, moghaddam_new_2011, shetty_can_2015, biedermann_release_2012}, (2) high selectivities between structurally related guests which translate into large $\Delta \Delta G$ values~\cite{isaacs_stimuli_2014}, (3) low molecular weights (1--2 kDa) permitting high levels of theory to be used, and (4) highly restricted conformational degrees of freedom, reducing conformational sampling challenges often seen in protein-ligand binding.
-For SAMPL6-10, we will resynthesize a series of CB[$n$]-type receptors of increasing complexity, measure $K_a$ values, and determine host-guest stoichiometry and geometry toward pharmaceutically relevant guests (selected drugs) in order to stringently test methods for predicting binding.  
-Figure~\ref{figure:CB} shows the chemical structures of three hosts---Me$_4$CB[8]~\cite{vinciguerra_synthesis_2015}, glycoluril hexamer~\cite{lucas_templated_2011}, and acyclic CB[$n$]-type receptors~\cite{ma_acyclic_2012, ma_acyclic_2012-1, zhang_acyclic_2014, gilberg_acyclic_2015, sigwalt_acyclic_2016, zhang_acyclic_2014-1} which span a range in terms of level of preorganization and formal charge.
-
 \begin{figure}[h]
 \begin{centering}
 \resizebox{\textwidth}{!}{\includegraphics{figures/CB.pdf}}
 
 \end{centering}
 
 \vspace{0.1in}
-\caption{\label{figure:CB} \footnotesize {\bf Structures of Me$_4$CB[8], glycouril hexamer, and acyclic CB[n]-type receptors.} These receptors bind a variety of drug-like molecules, some with high affinity, and this binding will form the basis of a cucubituril-based series of SAMPL binding challenges.}
+\caption{\label{figure:CB} \footnotesize {\bf Structures of Me$_4$CB[8], glycouril hexamer, and acyclic CB[n]-type receptors.} These receptors bind a variety of drug-like molecules, some with high affinity, and this binding will form the basis of the SAMPL6-10 binding challenges.}
```

Comment:
Change bold part to "SAMPL6-10 host-guest challenges will feature cucurbituril hosts, including Me$_4$CB[8], glycouril hexamer, and acyclic CB[n]-type receptors."

Also can use `\vspace{-0.2in}` to close up spaces above and below caption.